packages/components/Callout/macos/CalloutView.swift

```diff
@@ -26,6 +26,8 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 		}
 	}
 
+    @objc public var setInitialFocus: Bool = false
+
 	@objc public var onShow: RCTDirectEventBlock?
```

Comment:
does this need to be indented to match the other variables

Reply:
Also some indents below don't seem right

Reply:
Yeah it's a tabs vs spaces thing. Good catch.